### PR TITLE
Fix website build

### PR DESCRIPTION
**Pull request by Michael1993**

### CODE REVIEW

**Comment by Michael1993** [comment]
https://github.com/junit-pioneer/junit-pioneer.github.io/actions/runs/868879285

Looks to me this fixed the `buildSite` command.

File: build.gradle [hunk 1]
```diff
@@ -116,9 +116,12 @@ task insertVersion() {
 }
 
 def insertProjectVersion(String project, String abbreviation) {
-	Properties properties = new Properties()
-	file("${project}/version.properties").withInputStream { properties.load(it) }
-	String version = properties.previousVersion
+	final def stdout = new ByteArrayOutputStream()
```
Comment (aepfli) [inline build.gradle]: i know we are using a lot of command line thingies execution already, but, maybe grgit would be a nice option here - http://ajoberstar.org/grgit/grgit-tag.html 

but it looks fine form my side

**Review by aepfli — APPROVED** [review]
lgtm, but i would switch to grgit instead of command line

File: build.gradle (hunk 1 above)
Comment (Michael1993) [reply build.gradle]: I'll look into it. Soon™.

**Comment by Michael1993** [comment]
https://github.com/junit-pioneer/junit-pioneer.github.io/actions/runs/921135528

Works fine with `grgit`

File: build.gradle (hunk 1 above)
Comment (Michael1993) [reply build.gradle]: Looked into it. Implemented it.

**Review by aepfli — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -115,10 +119,9 @@ task insertVersion() {
 	}
 }
 
-def insertProjectVersion(String project, String abbreviation) {
-	Properties properties = new Properties()
-	file("${project}/version.properties").withInputStream { properties.load(it) }
-	String version = properties.previousVersion
+def insertProjectVersion(final String project, String abbreviation) {
+	final def tags = grgit.open(dir: project).tag.list().collect { it.getName() }
+	final def version = tags.max()
```
Comment (beatngu13) [inline build.gradle]: Three notes:

1. Below we are explicit about the types (`File`, `String`, …), here you use `def` (and `final`). Personally, I would prefer a consistent style.
2. I like the "one op per line" style:

```groovy
final def tags = grgit.open(dir: project)
        .tag
        .list()
        .collect { it.getName() }
```
3. Not that familiar with Gradle-Groovy, but would a method reference also work for `{ it.getName() }`?

**Review by beatngu13 — APPROVED** [review]
Since my three notes are more or less optional, I approve and leave it up to you, if you want to adopt something or merge immediately. 😉✌️

File: build.gradle (hunk 2 above)
Comment (nipafx) [reply build.gradle]: 1. Yes
2. Yes
3. that code was removed

**Review by nipafx — APPROVED** [review]
LGTM, will merge.

**Pull request merged**